src/objects/Error_Settings__c.object

```diff
@@ -70,6 +70,15 @@ To view these jobs, search for Scheduled Jobs in Salesforce Setup.</description>
         <type>TextArea</type>
     </fields>
     <fields>
+        <fullName>Sandbox_Jobs_Rescheduled__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Used to track when NPSP apex jobs have been deleted and rescheduled in a Sandbox. It should remain unchecked in all Production orgs</description>
```

Comment:
I was thinking that since this field should never be visible/modifiable directly by end users the string/label wouldn't necessarily matter. What do you think @smarovich?